Sources/LanguageServerProtocol/SupportTypes/InlayHint.swift

```diff
@@ -16,33 +16,120 @@ public struct InlayHint: ResponseType, Codable, Hashable {
   public var position: Position
 
   /// The hint's kind, used for more flexible client-side styling.
-  public let category: InlayHintCategory?
+  public let kind: InlayHintKind?
 
   /// The hint's text, e.g. a printed type
-  public let label: String
+  public let label: InlayHintLabel
+
+  /// Optional text edits that are performed when accepting this inlay hint.
+  public let textEdits: [TextEdit]?
+
+  /// The tooltip text displayed when the inlay hint is hovered.
+  public let tooltip: MarkupContent?
+
+  /// Whether to render padding before the hint.
+  public let paddingLeft: Bool?
+
+  /// Whether to render padding after the hint.
+  public let paddingRight: Bool?
 
```

Comment:
I think you’re missing the `data?: LSPAny` field that’s defined in the spec here.

Reply:
Good catch, thanks!